src/main/java/com/aws/greengrass/certificatemanager/certificate/CertificatesConfig.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.certificatemanager.certificate;
+
+import com.aws.greengrass.componentmanager.KernelConfigResolver;
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.util.Coerce;
+
+public class CertificatesConfig {
+    static final int MAX_SERVER_CERT_EXPIRY_SECONDS = 60 * 60 * 24 * 10; // 10 days
+    static final int DEFAULT_SERVER_CERT_EXPIRY_SECONDS = 60 * 60 * 24 * 7; // 7 days
+    static final int DEFAULT_CLIENT_CERT_EXPIRY_SECONDS = 60 * 60 * 24 * 7; // 7 days
+
```

Comment:
add the config for pre-expiration rotation too? With this class added it shouldn't be too difficult.

Reply:
We'll be ready for it if/when we add it. I'm not sold on whether that's the correct approach yet, so will hold off for now